communities/massmesh/meshnode/files/etc/config/system

```diff
@@ -0,0 +1,19 @@
+config system
+    option hostname 'OpenWrt'
+    option ttylogin '1'
+    option log_size '64'
+    option urandom_seed '1'
```

Comment:
toggled this.. interesting stuff. looks like a lengthy first boot, but going forward boots are seeded. begs question shoudl we create and include a pre-seed file during cook-time?